remote_command_execution_vulnerability.py

```diff
@@ -23,6 +23,7 @@
 import random
 import hashlib
 import platform
+import socket

```

Comment:
Does this need to be added to the [requirements.txt](https://github.com/acecilia/OpenWRTInvasion/blob/master/requirements.txt) file?